### PR TITLE
ref(workflow): Convert processing hint styles to emotion

### PR DESCRIPTION
**Pull request by scttcper**
Deleted the global styles and used our alert styles, the global styles seemed outdated already with the weird underlined button and old color scheme.
Adds stories for this component

Before
image

After
image

Mobile...
image

### CODE REVIEW

File: src/sentry/static/sentry/app/types/index.tsx [hunk 1]
```diff
@@ -736,7 +736,7 @@ export type ProcessingIssue = {
   numIssues: number;
   signedLink: string;
   lastSeen: string;
-  hasMoreResolvableIssues: boolean;
+  hasMoreResolveableIssues: boolean;
```
Comment (scttcper) [inline src/sentry/static/sentry/app/types/index.tsx:739]: backend has a comment where they're keeping the spelling mistake for legacy api use. I don't believe the frontend is using it.

Comment (billyvg) [reply src/sentry/static/sentry/app/types/index.tsx:739]: Hmmmm, doesn't that mean the frontend uses the backend API response?

Comment (scttcper) [reply src/sentry/static/sentry/app/types/index.tsx:739]: Correct, the frontend uses this api, but this property didn't match the backend's spelling

**Review by davidenwang — APPROVED** [review]

File: src/sentry/static/sentry/app/types/index.tsx (hunk 1 above)
Comment (billyvg) [reply src/sentry/static/sentry/app/types/index.tsx:739]: Ah I see

**Review by billyvg — APPROVED** [review]

**Pull request merged**